tests/ui/unwrap.stderr

```diff
@@ -8,12 +8,20 @@ LL |     let _ = opt.unwrap();
    = help: if you don't want to handle the `None` case gracefully, consider using `expect()` to provide a better panic message
 
 error: used `unwrap()` on `a Result` value
-  --> $DIR/unwrap.rs:10:13
+  --> $DIR/unwrap.rs:12:13
    |
 LL |     let _ = res.unwrap();
    |             ^^^^^^^^^^^^
    |
    = help: if you don't want to handle the `Err` case gracefully, consider using `expect()` to provide a better panic message
 
-error: aborting due to 2 previous errors
+error: used `unwrap()` on `an Option` value
+  --> $DIR/unwrap.rs:20:17
+   |
+LL |         let _ = opt.unwrap();
+   |                 ^^^^^^^^^^^^
+   |
+   = help: if you don't want to handle the `None` case gracefully, consider using `expect()` to provide a better panic message
```

Comment:
Wait, this shouldn't get linted 🤔

Reply:
Yeah, I'm debugging right now but stacking